components/Shared/Input/Text.jsx

```diff
@@ -38,7 +38,13 @@ const TextInput = ({
             {...props}
           />
           {denom && (
-            <DenomTag backgroundColor='core.transparent' height={7}>
+            <DenomTag
+              height={7}
+              css={`
+                background-color: ${props =>
```

Comment:
This is one of those "css override" things kyle was talking about. We should be able to just do:

```
<DenomTag
bg="core.transparent"
...
/>
```